checkov/common/runners/runner_registry.py

```diff
@@ -655,24 +654,25 @@ def enrich_report_with_guidelines(scan_report: Report) -> None:
     def get_enriched_resources(
         repo_roots: list[str | Path], download_external_modules: bool
     ) -> dict[str, dict[str, Any]]:
+        from checkov.terraform.modules.module_objects import TFDefinitionKey
```

Comment:
why adding here and not top level? you can also put it under a `if TYPE_CHECKING:` block